### ISSUE TITLE
Feature: Support docker run --pull=never

### ISSUE THREAD

**Issue by Hennns** [ansible-collections/community.docker#783]
##### SUMMARY

Currently the docker_container_module supports a boolean for the pull policy. 
Where pull: true is equivalent to using --pull=always
and pull: false is equivalent to using --pull=missing

However, there is a 3rd option -> --pull=never
The suggestion is to add support for never. 

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME

docker_container_module

##### ADDITIONAL INFORMATION

When starting containers on systems not connected to a repository (i.e it is expected that required images is already present, for example loaded from a file). it is confusing with an error that docker pull failed - when the real problem is that the required image was never loaded.
Similarly, it is confusing when a playbook (accidentally) works in an environment that is connected to a registry (say when developing/testing it) however then does not work in other environments (and then with a misleading error).
Being able to use --pull=never would ensure things never accidentally work. 

Example of expected behavior from docker docs: https://docs.docker.com/engine/reference/commandline/container_run/

```code
docker run --pull=never hello-world
docker: Error response from daemon: No such image: hello-world:latest.
```

**Comment by felixfontein** [comment]
This is a good idea. And again shows why using a boolean for a parameter can be bad choice :)

**Comment by felixfontein** [comment]
I've implemented this in #797.

**Comment by Hennns** [comment]
Thank you, and thank you for maintaining the community.docker collections!
